haas/rest.py

```diff
@@ -164,10 +170,13 @@ def foo(bar, baz, quux):
           whose second is an integer (the status code).
     """
     def register(f):
+
+        methods = method if isinstance(method, list) else [method]
```

Comment:
If I use the argument error from the parent function, I get the following error. Any ideas?

In the meantime, I'll create a version with an intermediate variable, though I do think python inline if saves a ton of space in instances like this and is the most "pythonic" way to do this.

```python
tests/unit/rest.py:33: in <module>
    from haas.test_common import config_testsuite
venv/local/lib/python2.7/site-packages/_pytest/assertion/rewrite.py:171: in load_module
    py.builtin.exec_(co, mod.__dict__)
haas/test_common.py:19: in <module>
    from haas import api, config
venv/local/lib/python2.7/site-packages/_pytest/assertion/rewrite.py:171: in load_module
    py.builtin.exec_(co, mod.__dict__)
haas/api.py:37: in <module>
    @rest_call('GET', '/projects', Schema({}))
haas/rest.py:174: in register
    if not isinstance(methods, list):
E   UnboundLocalError: local variable 'methods' referenced before assignment
```